edk2-build-toolchain/README.md

```diff
@@ -11,7 +11,7 @@ There are two ways to obtain the docker image:
 To build the container image, run the following command at the root of the project:
 
 ``` console
-docker build --tag dvuefi-build edk2-build-toolchain
+docker build --tag debian-qemu-uefi edk2-build-toolchain
```

Comment:
I think I would like to keep the name `dvuefi-build` here, as `debian-qemu-uefi` sounds misleading (at one point qemu was included in the image, but it isn't anymore).

Could we references to `debian-qemu-uefi` to be `dvuefi-build` instead?